cli/cmd/alert_channel.go

```diff
@@ -0,0 +1,227 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	alertChannelCommand = &cobra.Command{
+		Use:     "alert-channel",
+		Aliases: []string{"alert-channels", "ac"},
+		Short:   "Manage alert channels",
+		Long:    "Manage alert channels integrations with Lacework",
+	}
+
+	// alertChannelsListCmd represents the list sub-command inside the alert channels command
+	alertChannelListCmd = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all available alert channel integrations",
+		Args:    cobra.NoArgs,
+		RunE:    alertChannelList,
+	}
+
+	// alertChannelShowCmd represents the show sub-command inside the alert channel command
+	alertChannelShowCmd = &cobra.Command{
+		Use:     "show",
+		Aliases: []string{"get"},
+		Short:   "Show a single alert channel integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    alertChannelShow,
+	}
+
+	// alertChannelCreateCmd represents the show sub-command inside the alert channels command
+	alertChannelCreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "Create a new alert channel integration",
+		Args:  cobra.NoArgs,
+		RunE:  alertChannelCreate,
+	}
+
+	// alertChannelDeleteCmd represents the delete sub-command inside the alert channels command
+	alertChannelDeleteCmd = &cobra.Command{
+		Use:     "delete",
+		Aliases: []string{"rm"},
+		Short:   "Delete a alert channel integration",
+		Args:    cobra.ExactArgs(1),
+		RunE:    alertChannelDelete,
+	}
+)
+
+func init() {
+	// add the alert-channel command
+	rootCmd.AddCommand(alertChannelCommand)
+	alertChannelCommand.AddCommand(alertChannelListCmd)
+	alertChannelCommand.AddCommand(alertChannelShowCmd)
+	alertChannelCommand.AddCommand(alertChannelCreateCmd)
+	alertChannelCommand.AddCommand(alertChannelDeleteCmd)
+}
+
+func alertChannelsToTable(integrations []api.AlertChannelRaw) [][]string {
+	var out [][]string
+	for _, cadata := range integrations {
```

Comment:
this is also wrong, but since it is code, I will let it go. (should be alertChannels or something like that)